devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
+[VI] indicates either V or I).  These files can be used to initialize a
```

Comment:
The entries and ```IDENT``` in both catalogs correspond to the same galaxies. I made separate catalogs so that each could be used independently. Also  ```ChromaticRealGalaxies```  takes two separate ```RealGalaxyCatalog```s and an ```ident``` as input to create the chromatic galaxy. @jmeyers314 do you have any comments on this?